src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/whitespace/NoWhitespaceBeforeCaseDefaultColonCheck.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<checkstyle-metadata>
+   <module>
+      <check fully-qualified-name="com.puppycrawl.tools.checkstyle.checks.whitespace.NoWhitespaceBeforeCaseDefaultColonCheck"
+             name="NoWhitespaceBeforeCaseDefaultColon"
+             parent="com.puppycrawl.tools.checkstyle.TreeWalker">
+         <description>&lt;p&gt;
+ Checks that there is no whitespace before a colon token of a switch case or default.
```

Comment:
Just a thought:
`Checks that there is no whitespace before the colon in a switch label.`?
switch case or default == case label https://docs.oracle.com/javase/specs/jls/se7/html/jls-14.html#jls-14.11

Reply:
Done

Reply:
Resolved.